src/pages/VenuePage/VenuePage.tsx

```diff
@@ -228,7 +250,7 @@ export const VenuePage: React.FC = () => {
   const { template, hasPaidEvents } = space;
 
   const hasEntrance = !!world?.entrance?.length;
-  const hasEntered = enteredVenueIds?.includes(spaceId);
+  const hasEntered = world?.id && enteredWorldIds?.includes(world.id);
```

Comment:
Expect to have an array with `null` or `undefined` values in it?

Reply:
`enteredWorldIds` might not be present on the owning document. If it exists it should only have actual world IDs though :)